src/EditorFeatures/TestUtilities/Classification/FormattedClassification.cs

```diff
@@ -41,11 +41,15 @@ public override string ToString()
                     switch (Text)
                     {
                         case "(":
-                            return "Punctation.OpenParen";
+                            return "Punctuation.OpenParen";
```

Comment:
so this didn't break any existing tests?

Reply:
AFAICT, this is only used to generate helper strings in case a test fails.